src/app/tasks/create-task/create-task.component.html

```diff
@@ -6,29 +6,57 @@
             </div>
             <div class="panel-body">
                 <form [formGroup]="createTaskForm" (ngSubmit)="onSubmit()">
-                    <label for="title">Task title</label><br>
-                    <input type="text" id="title" name="title" formControlName="title"><br>
-                    @if (createTaskForm.get('title')?.errors) {
-                        @if (createTaskForm.get('title')?.errors!['required'] && createTaskForm.get('title')?.touched) {
-                            <span class="help-block">Title is required</span>
+                    <div class="form-group row">
+                        <label class="col-sm-2 col-form-label" for="title">Task title</label>
+                        <div class="col-sm-10">
+                            <input class="form-control" type="text" id="title" name="title" formControlName="title">
+                            @if (createTaskForm.get('title')?.errors) {
+                                @if (createTaskForm.get('title')?.errors!['required'] && createTaskForm.get('title')?.touched) {
```

Comment:
non null assertion

Reply:
Is non-null assertion missing or should I replace the optional operator with the non-null assertion? When I remove it from the 'errors' attribute I get an error, that it could be null.

Reply:
![image](https://github.com/TomsBemis/task-manager-frontend/assets/107789468/80306328-f422-4053-93b4-acb304a97333)